README.md

```diff
@@ -36,6 +36,8 @@ npm run dev
 ## Documentation
 - To get the documention of rest apis import `Open Certs.postman_collection.json` into Postman.
 
+## Github setup

```

Comment:
Put `Github setup` before `Documentation` it makes more sense.

Reply:
done